content/en/docs/instrumentation/java/automatic/spring-boot.md

```diff
@@ -54,5 +54,78 @@ dependencies {
 
 ## Additional instrumentations
 
-You can configure additional instrumentations with
+### JDBC Instrumentation
+
+To have JDBC instrumentation with the OpenTelemetry Spring starter, add the
+following dependency:
+
+{{< tabpane text=true >}} {{% tab header="Maven (`pom.xml`)" lang=Maven %}}
+
+```xml
+<dependencies>
+	<dependency>
+		<groupId>io.opentelemetry.instrumentation</groupId>
+		<artifactId>opentelemetry-jdbc</artifactId>
+		<version>{{% param vers.instrumentation %}}</version>
```

Comment:
is this also `-alpha`?

Reply:
It should be -aplha, like the other dependencies of this page. I have fixed this typo. Thanks!